### PR TITLE
Align with documentation

### PR DESCRIPTION
**Pull request by andresrinivasan**
Align with [documentation](https://oss.redislabs.com/redisai/quickstart/#building-the-module).

Signed-off-by: André Srinivasan <andre@redislabs.com>

### CODE REVIEW

**Comment by filipecosta90** [comment]
@andresrinivasan I believe we need to update our Readme with:
```
make -C opt build
```
as we've updated the documentation: https://oss.redislabs.com/redisai/quickstart/#building-the-module 

@rafie @lantiga please advise :)

**Comment by andresrinivasan** [comment]
Corrected README and fixed bug in `opt/Makefile`.

File: opt/Makefile [hunk 1]
```diff
@@ -156,7 +156,7 @@ clean:
 ifeq ($(ALL),1)
 	$(SHOW)if [ -d "$(BINROOT)" ]; then rm -rf $(BINROOT); fi
 	$(SHOW)if [ -d "$(INSTALL_DIR)" ]; then rm -rf $(INSTALL_DIR); fi
-	$(SHOW)rm -f $(ROOT)/install-$(DEVICE)
+	$(SHOW)rm -rf $(ROOT)/install-$(DEVICE)
```
Comment (lantiga) [inline opt/Makefile:159]: This (e.g. `install-gpu` or `install-cpu`) is supposed to be a symlink. Not recursing would suffice in that case. Did you incur in a situation where recursion was necessary?

Comment (andresrinivasan) [reply opt/Makefile]: Must have been an artifact from an earlier build. Started clean and I didn't need to recurse. Corrected PR.

**Comment by lantiga** [comment]
Thanks!

**Pull request merged**